test/com/gfredericks/test/chuck/clojure_test_test.cljc

```diff
@@ -15,13 +15,13 @@
   ;; no option is OK, defaults to 100 tests
   (let [nb-runs (atom 0)]
     (checking "no option works" [i gen/s-pos-int]
-              (swap! nb-runs inc)
-              (is (> 0)))
+      (swap! nb-runs inc)
+      (is (> 0)))
```

Comment:
this isn't really core to the test, but I assume you meant `(> i 0)` or `(pos? i)` here?